benchflow-test-manager/application/src/main/java/cloud/benchflow/testmanager/strategy/selection/ExperimentSelectionStrategy.java

```diff
@@ -0,0 +1,13 @@
+package cloud.benchflow.testmanager.strategy.selection;
+
+/**
+ * @author Jesper Findahl (jesper.findahl@usi.ch)
+ *         created on 2017-04-20
+ */
+public interface ExperimentSelectionStrategy {
```

Comment:
We copy some of the information from the DSL model to the Test Model so that we store only the data we need as part of the Test Manager model.